core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -3933,6 +3947,10 @@ private static String javadocLink(String fullyQualifiedClassname) {
         "alluxio.master.embedded.journal.shutdown.timeout";
     public static final String MASTER_EMBEDDED_JOURNAL_TRIGGERED_SNAPSHOT_WAIT_TIMEOUT =
         "alluxio.master.embedded.journal.triggered.snapshot.wait.timeout";
+    public static final String MASTER_EMBEDDED_JOURNAL_TRANSPORT_REQUEST_TIMEOUT_MS =
```

Comment:
also move `MASTER_EMBEDDED_JOURNAL_TRANSPORT_REQUEST_TIMEOUT` before `MASTER_EMBEDDED_JOURNAL_TRIGGERED_SNAPSHOT_WAIT_TIMEOUT ` to keep them sorted?